src/SFA.DAS.FindEpao.Api.UnitTests/Controllers/Courses/WhenGettingCourseList.cs

```diff
@@ -35,7 +36,8 @@ public class WhenGettingCourseList
             controllerResult.StatusCode.Should().Be((int)HttpStatusCode.OK);
             var model = controllerResult.Value as GetCourseListResponse;
             Assert.IsNotNull(model);
-            model.Courses.Should().BeEquivalentTo(mediatorResult.Courses);
+            model.Courses.Should().BeEquivalentTo(mediatorResult.Courses, options => options.ExcludingMissingMembers());
```

Comment:
See previous comment on `ExcludingMissingMembers`

Reply:
As above, switched to explicits.